bin/test-dag-tags.py

```diff
@@ -0,0 +1,110 @@
+#!/bin/python3
+from collections import defaultdict
+import subprocess
+from typing import Dict
+
+
+VALID_TAGS = ("impact/tier_1", "impact/tier_2", "impact/tier_3", "repo/bigquery-etl", "repo/telemetry-airflow",)
+REQUIRED_TAG_TYPES = ("impact",)
+
+
+class DagValidationError(Exception):
+    pass
+
+
+def _execute_shell_cmd(cmd: str, cmd_params: Dict[str, dict] = dict()) -> str:
+    mask_string = "###"
+
+    cmd_params_log = {
+        key: val["value"] if not val["value"] else mask_string
+        for key, val in cmd_params.items()
+    }
+
+    print("Executing command: %s" % (cmd.format(**cmd_params_log)))
+
+    cmd_params_formatted = {
+        key: val["value"]
+        for key, val in cmd_params.items()
+    }
+
+    cmd_output = subprocess.run(
+        cmd.format(**cmd_params_formatted),
+        shell=True,
+        capture_output=True,
+        text=True,
+    )
+    try:
+        cmd_output.check_returncode()
+    except subprocess.CalledProcessError as _err:
+        print(cmd_output.stdout)
+        print(cmd_output.stderr)
+        raise _err
+
+    print("Command executed successfully, processing output...")
+
+    return cmd_output.stdout.strip().replace("\r", "").split("\n")
+
+
+def retrieve_existing_airflow_dags_from_db(pswd: str) -> Dict[str, str]:
+    shell_cmd = "docker-compose exec web mysql -Ns -h db -u root -p{pswd} airflow -e 'SELECT dag_id, name FROM dag_tag;'"
+    cmd_params = {
+        "pswd": {
+            "value": pswd,
+            "is_sensitive": True,
+        }
+    }
+
+    cmd_output = _execute_shell_cmd(shell_cmd, cmd_params)
+
+    dags = defaultdict(list)
+
+    for dag in cmd_output:
+        dag_name, tag_name = dag.split("\t")
+        dags[dag_name] = dags[dag_name] + [tag_name]
+
+    print("Number of DAGs with tags found: %s" % (len(dags)))
+
+    return dags
+
+
+def get_number_of_dags_loaded(pswd) -> int:
+    cmd_params = {
+        "pswd": {
+            "value": pswd,
+            "is_sensitive": True,
+        }
+    }
+    return _execute_shell_cmd("docker-compose exec web mysql -Ns -h db -u root -p{pswd} airflow -e 'SELECT dag_id from dag WHERE is_subdag = 0;'", cmd_params)
+
+
+if __name__ == "__main__":
+    db_pass = "secret"
+    # Assumes the web and db containers are already running
+    tag_errors = 0
+
+    dags = get_number_of_dags_loaded(db_pass)
```

Comment:
nit: func name implies it returns length. change name to something like get_loaded_dags_from_sql

Reply:
that's a fair point, I believe originally that's what it did then I changed it and forgot to update the fun name. Fixed that now.